robottelo/api/utils.py

```diff
@@ -213,3 +217,78 @@ def get_role_by_bz(bz_id):
                 ]
             entities.Filter(permission=permissions, role=role).create()
     return role.read()
+
+
+def create_role_permissions(role, permissions_types_names):
```

Comment:
Just a note, as this function deals with API calls, and for now we are not going to cover this functionality in our unit tests (we can do that one day when we have API mock) but by now we can add:

```python
def create_role_permissions(role, permissions_types_names):  # pragma: no cover
```

When it is time, we can review all `pragma: no cover` and implement an API mock to have more tests.